src/main/java/org/folio/circulation/services/PubSubPublishingService.java

```diff
@@ -66,4 +69,42 @@ public PubSubPublishingService(RoutingContext routingContext) {
 
     return publishResult;
   }
+
+  /**
+   * Method that publishes event to PubSub and includes response body in the exception message in
+   * case of any response status other than 204.
+   *
+   * @param eventMessage Event to be published
+   * @param params Okapi connection parameters
+   * @return
+   */
+  private CompletableFuture<Boolean> sendEventMessage(Event eventMessage,
+    OkapiConnectionParams params) {
+
+    PubsubClient client = new PubsubClient(params.getOkapiUrl(), params.getTenantId(),
```

Comment:
Is it possible to declare the client as a class member?

Reply:
Done